game/static/game/css/404.css

```
@@ -0,0 +1,124 @@
+    box-shadow: 0 22px 50px rgba(0, 0, 0, 0.38);
+}
+
+.crest {
+    width: 70px;
+    height: auto;
+    margin-bottom: 0.85rem;
+    filter: brightness(1.5) contrast(1.2) drop-shadow(0 0 8px rgba(0, 200, 255, 0.35));
+}
+
+.code {
+    font-family: 'Cinzel', serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)